volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -635,6 +635,20 @@ def get_create_time(self) -> datetime.datetime:
         # root time namespace, not within the task's own time namespace
         return boottime + task_start_time_timedelta
 
+    def get_parent_pid(self) -> int:
```

Comment:
I think technically this is an additive change, and there's no versioning on structs, so it's probably a MINOR bump on the framework version please?  That'll also mean all the required_framework_versions will need upping to whatever this ends up being (2.13.0 mebbe?)